docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,102 @@
+# Hotplug Network Interfaces
+KubeVirt now supports hotplugging network interfaces into a running Virtual
+Machine Instance (VMI). The hotplugged interface's model  will be `virtio` and
+it will be of type [bridge](http://kubevirt.io/api-reference/main/definitions.html#_v1_interfacebridge).
+
+When a VM having hotplugged interfaces is rebooted, the hotplugged interfaces
+will be attached to the restarted VM. If these interfaces are persisted, they
+become part of the VM spec - i.e. will not be considered hotplugged.
+
+## Enabling network interface hotplug support
+Network interface hotplug support must be enabled via a
+[feature gate](https://kubevirt.io/user-guide/operations/activating_feature_gates/#how-to-activate-a-feature-gate).
+The feature gates array in the KubeVirt CR must feature `HotplugNICs`.
```

Comment:
Would you please document the command patching KubeVirt to add the gate? I'd like this guide to be standalone, so we can just run through it, copy-pasting.